smg/normalise_location.py

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env python3
+
+import re
+import time
+
+from multiprocessing import Pool
+
+import geopy
+import pandas as pd
+
+
+GOOGLE_API_KEY = "AIzaSyDtcYWNxzjjZRZrkPPAxEgrGD78Ey7pc50"
```

Comment:
Cheers. Revoked key, we'll need to replace it if we're to make any future requests to the API.